AwsCryptographicMaterialProviders/runtimes/python/pyproject.toml

```
@@ -8,7 +8,7 @@ packages = [
 ]
 # Include generated internaldafny files in package distributions,
 # even though they are not included in version control
-include = ["**/internaldafny/generated/*.py"]
+include = ["src/aws_cryptographic_material_providers/internaldafny/generated/*.py"]
```

Curious if this works?

```suggestion
include = ["src/**/internaldafny/generated/*.py"]
```

I'd prefer that as it's more copy-paste-able.
If not, this is fine.